contents.json

```diff
@@ -5708,6 +5708,12 @@
         "description": "Efficient Unix file management.",
         "homepage": "https://github.com/dduan/Pathos",
         "tags": ["linux"]
+    }, {
+	"title": "VKPinCodeView",
+        "category": "textfield",
+        "description": "Simple and elegant UI component for input PIN",
```

Comment:
@Sunspension Hi! Take a look at this point "Start the description with a capital and end with a full stop/period." in the [contribution guidelines](https://github.com/matteocrippa/awesome-swift/blob/master/.github/CONTRIBUTING.md). 

Could you fix it?

Reply:
Sure, I did it, could you check it?